src/Notify/Resources/doc/index.rst

```diff
@@ -0,0 +1,140 @@
+Symfony UX Notify
+===================
+
+Symfony UX Notify is a Symfony bundle providing light dropzones for
+file inputs in Symfony Forms. It is part of `the Symfony UX initiative`_.
+
+Symfony UX Notify is a Symfony bundle integrating realtime native
+notifications in Symfony applications using `Mercure`_.
+It is part of `the Symfony UX initiative`_.
+
+Installation
+------------
+
+Before you start, make sure you have `Symfony UX configured in your app`_.
+
+Then, install this bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-notify
+
+    # Don't forget to install the JavaScript dependencies as well and compile
+    $ yarn install --force
+    $ yarn encore dev
+
+Also make sure you have at least version 3.0 of
+`@symfony/stimulus-bridge`_ in your ``package.json`` file.
+
+Configuration
+-------------
+
+Symfony UX Notify can be configured to specify the Mercure hub to use:
+
+.. code-block:: yaml
+
+    # config/packages/notify.yaml
+
+    notify:
+        # Specifies the Mercure hub to use. Defaults to "mercure.hub.default"
+        mercure_hub: mercure.hub.custom
+
+
+Usage
+-----
+
+To use Symfony UX Notify you must have a `running Mercure server`_.
+
+Then, you can inject the ``NotifierInterface`` service and send messages on the ``chat/mercure`` channel::
+
+    // ...
+    use Symfony\Component\Notifier\Notification\Notification;
+    use Symfony\Component\Notifier\NotifierInterface;
+
+    class AnnounceFlashSalesCommand extends Command
+    {
+        protected static $defaultName = 'app:flash-sales:announce';
+        private $notifier;
+
+        public function __construct(NotifierInterface $notifier)
+        {
+            parent::__construct();
+
+            $this->notifier = $notifier;
+        }
+
+        protected function execute(InputInterface $input, OutputInterface $output): int
+        {
+            $this->notifier->send(new Notification('Flash sales has been started!', ['chat/mercure']));
+
+            return 0;
+        }
+    }
+
+Finally, native notifications could be displayed using the ``stream_notifications`` Twig function:
+
+.. code-block:: twig
+
+    {{ stream_notifications(['/my/topic/1', '/my/topic/2']) }}
+    {{ stream_notifications() }}
+
+.. note::
+   *Note*: Calling ``stream_notifications`` without parameter will fallback to the
+   following unique topic: ``https://symfony.com/notifier``.
+
+Extend the default behavior
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony UX Notify allows you to extend its default behavior using a
+custom Stimulus controller:
+
+.. code-block:: javascript
+
+    // mynotify_controller.js
+
+    import { Controller } from '@hotwired/stimulus';
+
+    export default class extends Controller {
+        connect() {
+            this.element.addEventListener('notify:connect', this._onConnect);
```

Comment:
I hate to have to worry about it, but with this example, users are going to have the `this` problem down in `_onConnect()`. So perhaps we should add:

```js
initialize() {
    // guarantees "this" refers to this object in _onConnect
    this._onConnect = this._onConnect.bind(this);
}
```